### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by beklapko**
These changes are all for the readme file. I can add more images if needed but I thought what was there already was sufficient. I think the only step that's not covered in much detail is the "create a bot" step but I don't know if there are generic steps that we could add anyway. I'm also not sure if there's anything more we can add for the "Testing" and "Development" sections.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -8,94 +8,103 @@ if Harbormaster is not enabled).
...
 
-Fill in your "Phabricator URL" with the base
-URL of your phabricator install, for example `https://phabricator.example.com`.
-Enter the conduit token for your Jenkins bot user (create one if necessary). Add a
-Description for readability.
+Jenkins Setup
+-------------
 
-![Configure Credentials](/docs/configure-credentials.png)
+1. Navigate to `https://ci.example.com/configure` with your base Jenkins URL in place of "ci.example".
```
Comment (sectioneight) [inline readme.md]: nit:

in place of "ci.example.com"

File: readme.md [hunk 2]
```diff
@@ -8,94 +8,103 @@ if Harbormaster is not enabled).
...
-`https://ci.example.com/buildByToken/buildWithParameters?job=test-example&DIFF_ID=${buildable.diff}&PHID=${target.phid}`,
-replacing `https://ci.example.com` with the URI of your Jenkins instance, and
-`test-example` with the name of your job. If your Jenkins instance is exposed to
-the internet, make sure to install the [Build Token Root Plugin][] and fill in
-the `token` parameter.
-
-Set the "When Complete" dropdown to "Wait For Message"
-
-![Harbormaster plan](/docs/harbormaster-plan.png)
+1. Navigate to `https://phabricator.example.com/harbormaster/plan/` with your base Phabricator URL in place of `phabricator.example`.
```
Comment (sectioneight) [inline readme.md]: phabricator.example.com -- not everybody will be on a ".com" address

**Comment by sectioneight** [comment]
This looks good! Two small comments. Why is the PR called "Update Readme 1 20"

**Comment by beklapko** [comment]
I named my branch a little funny because I wasn't sure I did things correctly the first time. There should be underscores in that name: "update_readme_1_20" with "1_20" being yesterday's date.

**Comment by sectioneight** [comment]
Merged. Thanks, Brody!

**Pull request merged**